modules/ref_fb_module/src/renderer_fb_impl.cpp

```diff
@@ -174,9 +174,13 @@ void RendererFbImpl::readProperties()
     LOG_T("Properties: Custom2dMaxRange {}", custom2dMaxRange);
     if (custom2dMinRange > custom2dMaxRange)
     {
-        setComponentErrorStateStatusWithMessage(ComponentErrorState::Error,
-                                                "Property custom2dMaxRange have to be more then custom2dMinRange");
-        LOG_E("Property custom2dMaxRange have to be more then custom2dMinRange");
+        setComponentStatusWithMessage(ComponentStatus::Error,
+                                                "Property custom2dMaxRange has to be more than custom2dMinRange");
+        LOG_E("Property custom2dMaxRange has to be more than custom2dMinRange");
+    }
+    else
+    {
+        setComponentStatus(ComponentStatus::Ok);
```

Comment:
This might not be the best pattern for log handling, as we set the status to OK no matter the previous state. Likely the logs should be cleared in a separate method that checks all preconditions. 

This pattern might be okay in function blocks where the status is changed only within 1 function.